docs/source/deployment/security_checklist.md

```diff
@@ -0,0 +1,696 @@
+- Users cannot access the environment without MFA.
+- Users are required/advised to create passwords of a certain strength.
+
+### Implication:
+
+- Users are required to authenticate with Multi-factor Authentication (MFA) in order to access the secure analysis environment.
+- Passwords are strong
+
+### Verify by:
+
+#### Check: Non-group user cannot access the apps
+
+Attempt to login to the remote desktop web client as the research user.
+
+````{attention}
+{{camera}} <b>Verify that:</b>
+ <details><summary> user is prompted to setup MFA</summary>
+
+```{image} security_checklist/login_no_mfa_guacamole.png
+:alt: Guacamole MFA setup prompt
+:align: center
+```
+
+</details>
+````
+
+#### Check: Membership of the correct group is insufficient to give access
+
+[Assign the research user to SRE A](../management/index.md#assign-existing-users-to-an-sre).
+Again, attempt to login to the remote desktop web client.
+
+````{attention}
+{{camera}} <b>Verify that:</b>
+<details><summary>user is prompted to setup MFA</summary>
+
+```{image} security_checklist/login_no_mfa_guacamole.png
+:alt: Guacamole MFA setup prompt
+:align: center
+```
+
+</details>
+````
+
+#### User can self-register for MFA
+
+Check that the reserach user is able to successfully set up MFA.
+
+- Visit [`https://aka.ms/mfasetup`](https://aka.ms/mfasetup).
+- Login as the research user.
+
+```{attention}
+{{white_check_mark}} Verify that: user is guided to set up MFA
+```
+
+- Set up MFA as per {ref}`the user guide instructions <roles_researcher_user_guide_setup_mfa>`.
+
+````{attention}
+{{camera}} <b>Verify that:</b>
+
+<details><summary>MFA setup is successful</summary>
+
+```{image} security_checklist/aad_additional_security_verification.png
+:alt: AAD additional security verification
+:align: center
+```
+</details>
+````
+
+#### User can login after setting up MFA
+
+Check that the research user can authenticate using MFA and is granted access to the SRE.
+
+- Login to the remote desktop web client as the research user.
+
+````{attention}
+{{camera}} <b>Verify that:</b>
+<details><summary>you are prompted for MFA and can respond</summary>
+
+```{image} security_checklist/aad_mfa_approve_signin_request.png
+:alt: AAD MFA approve sign-in request
+:align: center
+```
+
+</details>
+````
+
+#### Authenticated user can access workspaces
+
+Check that the research user can access a workspace.
+
+- Login to the remote desktop web client as the research user.
+- Select a workspace and login as the research user.
+
+````{attention}
+{{camera}} <b>Verify that:</b>
+<details><summary>you can connect to <i>Desktop: Ubuntu0</i></summary>
+
+```{image} security_checklist/guacamole_srd_desktop.png
+:alt: SRD desktop
+:align: center
+```
+
+</details>
+````
+
+## 2. Isolated Network
+
+### Turing configuration setting:
+
+- {ref}`Researchers <role_researcher>` cannot access any part of an SRE (except the portal) from outside the SRE network.
```

Comment:
Does this mean the dashboard? Or the Azure portal? I don't think either of these should be accessible from outside an approved network.

Reply:
The Guacamole dashboard. "outside the SRE network" means "outside of the SRE vnet".

Or, another way of saying it would be "The Guacamole dashboard is the only part of the SRE exposed to the internet."

Reply:
Updated in 06b0938